pandas/core/algorithms.py

```diff
@@ -1102,6 +1102,27 @@ def _get_score(at):
         return result
 
 
+def check_percentile(q: Union[float, Iterable[float]]) -> np.ndarray:
+    """
+    Validate percentiles (used by describe and quantile).
+    Args:
```

Comment:
Just as an FYI this technically isn't a valid docstring as it doesn't follow the docstring guide:

https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#docstring

Though since not part of the public API I suppose not failing CI. Would be nice to update if you have to make other changes

Reply:
Thanks for the suggestion, I did not know this. I'll update the doctoring.